src/variables.js

```diff
@@ -146,7 +145,7 @@ module.exports = {
 				if (JSON.stringify(oldValue) !== JSON.stringify(newValue)) {
 					this.store[key] = newValue
 					if (key === 'clips') {
-						this.updateVariable('clip_name', this.store.clips[this.store.clipID].name)
+						this.updateVariable('clip_name', this.store.clips[this.store.clipID - 1]?.name)
```

Comment:
correction of clip index which displays the wrong clip name for the selected clip id.